templates/kubeconfig.tmpl

```diff
@@ -0,0 +1,25 @@
+apiVersion: v1
+clusters:
+- cluster:
+    certificate-authority: <PATH TO CA File>
```

Comment:
Can you update this to base64 encode and embed this cert? 
It would look like:

```
- cluster:
    certificate-authority-data: LS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0tLS0tCk1JSUM1ekNDQWMrZ0F3SUJBZ0lCQVRBTkJna3Foa2lHOXcwQkFRc0ZBREFWTVJNd0VRWURWUVFERXdwdGFXNXAKYTNWaVpVTkJNQjRYRFRFNE1ETXlPVEU1TURnd09Wb1hEVEk0TURNeU5qRTVNRGd3T1Zvd0ZURVRNQkVHQTFVRQpBeE1LYldsdWFXdDFZbVZEUVRDQ0FTSXdEUVlKS29aSWh2Y05BUUVCQlFBRGdnRVBBRENDQVFvQ2dnRUJBTHdHCkZySzZaZ3pBTVB0bGpOdVJZeWRmczhrRlF5Y1poTmV3UUF1NCtYV1dSOW1WM0wvMnBVS0pYbWMzdTQ1L1pkK2gKd2VRTUlFR0Z6LzVqalpVU0J4UjNVdTc3WlhFWWVRdEExU24rc2ZBVldKZ1FqNjFLQjdrdndLZ2xYY1E5QUlQYgpaYlZUa0pmWFEyeTA4YmhPck90eERkdExOeXhqbWhiY3g3UDl2SGRUMkxsMHVNaWpaVkRkaEJ5Q2FFY0Erdnl6Ckd2cDVjYmd5RjBWY2VvU3JTMmlEcnNyL1c3dXdaSWpYbGJoUitQNXBRZXVIc2dOQ1U3QXlyV3hPV1NwOGxlT2QKQ1VpeGxzNmpNdW4vZzFzSGVBS3Z6Vkl2REFROW9KcFNGU2wraC9KTnRWZFpBdlNYRkpNbmxKTGxIMWpOU0pSVgo4cFd3VGVlNDBOc0sxcjdmbG5NQ0F3RUFBYU5DTUVBd0RnWURWUjBQQVFIL0JBUURBZ0trTUIwR0ExVWRKUVFXCk1CUUdDQ3NHQVFVRkJ3TUNCZ2dyQmdFRkJRY0RBVEFQQmdOVkhSTUJBZjhFQlRBREFRSC9NQTBHQ1NxR1NJYjMKRFFFQkN3VUFBNElCQVFDNTY5cVh0R2s5UldkV09zQ3BjZnlmdTZBK0oyZVgvVDViRFQvS1RJaXVSVnNrbnpyUgpCWktjUG9UWWFxdmNMQVlONXMxSkVtbmJEanZ2WDZuM0R1NGtvUTkwUDNTT3BXbng2bFZZRzFPMUt1YzhEaEZCClhTZ0JNSkhFWkVVVm0yYlB1TEZ0RTdxcXZMQlJBZGFxSVhKUTRRcm9sM20wVGprT1BQUU1aNE5ZK294Um1ZK1QKZ3UzZEZRUnFDcVR4WUZuR2dzSHR4QjB2bkpVMTVhWmNqYytrZnA3RnB0OWZiNzg5Y1BXM0JqNWdBaURzZ2dMKwpDNnBJMVIwMXlqSUpPRU1kL3BGZTN5VHZJYTFjdDdyNklqenpvSmRRYVlSTmtXd0YzLzVxeTk2cDNNRDhVK05VCmFWek5pTW9RL0NaYVpnWHJhVm4rQ2VTblhDd0pyYis2eWRPZQotLS0tLUVORCBDRVJUSUZJQ0FURS0tLS0tCg==
```

Reply:
I've added this in [my fork](https://github.com/joerocklin/gangway/tree/kubeconfig-download) and submitted a PR to @jpweber to update his branch (to ultimately land back here).